src/Symfony/Component/Console/Output/ConsoleSectionOutput.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Output;
+
+use Symfony\Component\Console\Formatter\OutputFormatterInterface;
+use Symfony\Component\Console\Terminal;
+
+/**
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+class ConsoleSectionOutput extends StreamOutput
+{
+    public $content;
+    public $lines = 0;
+
+    private $sectionReference;
+    private $terminal;
+
+    /**
+     * @param resource                 $stream
+     * @param OutputSectionReference   $sectionReference
+     * @param int                      $verbosity
+     * @param null                     $decorated
+     * @param OutputFormatterInterface $formatter
```

Comment:
`OutputFormatterInterface|null`? null to use default OutputFormatter

Reply:
See reply below, we want to use the values from the original console output